src/sound.cpp

```diff
@@ -22,6 +22,11 @@
 #include "safeguards.h"
 
 static SoundEntry _original_sounds[ORIGINAL_SAMPLE_COUNT];
+/**
+ * The sound file for the original sounds, i.e. those not defined/overridden by a NewGRF.
+ * Needs to be kept alive during the game as _original_sounds[n].file refer to this.
+ */
+static std::unique_ptr<RandomAccessFile> _original_sound_file;
```

Comment:
I think I was unclear before

This variable is not used outside `OpenBankFile`. As such, there is no need that I can see for it to be global. `_original_sounds` refers to the memory that this variable points to, not the variable itself (which is reset every time the function is called anyway)

Reply:
I have moved it into the function and made it a static local variable?